### PR TITLE
[GH-#283] Adjusted transaction fields encoding

### LINKED ISSUE #283: adjust transaction fields encoding
dependend on #282 

- [x] hash as base58c "th"
- [x] signatures as list of base58c "sg"
- [x] sender as base58c "ak"
- [x] recipient as base58c "ak"

### PR DESCRIPTION
**Pull request by Artur64**
Resolves #283

### CODE REVIEW

File: apps/aeutil/lib/serialization.ex [hunk 1]
```diff
@@ -266,6 +266,12 @@ defmodule Aeutil.Serialization do
   end
 
   defp serialize_txs_info_to_json([h | t], acc) do
+    tx_hash =
```
Comment (thepiwo) [inline apps/aeutil/lib/serialization.ex]: either rename this, as it is not the hash or include hashing with a pipe

Comment (cytadela8) [reply apps/aeutil/lib/serialization.ex]: Also you should call some `hash_tx` function here so hashing is defined in one place. I think we have `SignedTx.hash_tx`

**Comment by thepiwo** [comment]
@cytadela8 can your re-review and merge?

**Comment by cytadela8** [comment]
@Artur64 please resolve conflicts

**Pull request merged**